server/services/philips-hue/lib/light/light.getLights.js

```diff
@@ -29,52 +29,20 @@ async function getLights() {
     const lights = await hueApi.lights.getAll();
 
     lights.forEach((philipsHueLight) => {
-      switch (philipsHueLight.modelid) {
-        case 'LCT007': // hue white & color 2nd generation
-        case 'LCT015': // hue white & color 4th generation
-        case 'LST002': // hue lightstrip indoor 2nd generation
-        case 'LCT024': // Hue play 1
-        case '440400982841': // Hue White and Color Ambiance Play Light Bar
-        case 'LCT010': // Hue A19 White & Color w/ Richer Colors
-        case 'LCA001': // Hue color lamp
-        case 'GL-C-008': // Non-hue LED Strip
-        case 'CLA60 TW OSRAM': // CLA60 TW OSRAM
-        case 'LLC020': // Hue go
-        case 'LCT012': // Hue White and Color Ambiance Candle E12
-        case 'LCT003': // Hue White and Color Ambiance Spot GU10
-        case 'LCL001': // Hue Lightstrip Plus
-        case 'LCS001': // Hue Lily Outdoor Spot Light RGBCCT
-        case 'RB 285 C': // Innr RGBW Bulb Colour E27
+      switch (philipsHueLight.type) {
+        case 'Extended color light':
           lightsToReturn.push(getPhilipsHueColorTemperatureLight(philipsHueLight, serialNumber, this.serviceId));
           break;
-        case 'LLC001': // LivingColor lamp 2nd generation
-        case 'LLC010': // Hue iris
+        case 'Color light':
           lightsToReturn.push(getPhilipsHueColorLight(philipsHueLight, serialNumber, this.serviceId));
           break;
-        case 'LWO001': // Hue White Filament Bulb G93 E27
-        case 'LWA001': // Hue White Bulb E27 (Dimmable light 2700K)
-        case 'LWB010': // Hue white bulb with fixed warming light (2700K)
-        case 'LWB006': // Hue white lamp
-        case 'LWG001': // Hue white spot 1
-        case 'LWG004': // Hue white spot
-        case 'LWV001': // Hue White Filament Bulb ST64 E27
-        case 'TRADFRI bulb E14 W op/ch 400lm': // IKEA white spot
-        case 'TRADFRI bulb E27 W opal 1000lm': // IKEA white lamp
-        case 'Classic A60 W clear - LIGHTIFY': // OSRAM Classic A60 W clear - LIGHTIFY
+        case 'Dimmable light':
           lightsToReturn.push(getPhilipsHueWhiteLight(philipsHueLight, serialNumber, this.serviceId));
           break;
-        case 'LTW012': // Hue White Ambiance E12
-        case 'LTW010': // Hue White & Ambiance Bulb
-        case 'LTW001': // Hue A19 White & Ambiance Bulb
-        case 'LTW013': // Hue White Ambiance GU10
-        case 'LTG002': // Hue White Ambiance GU10 w/ BT
-        case 'LTA001': // Hue White E27 with Bluetooth
+        case 'Color temperature light':
```

Comment:
color temperature = getPhilipsHueWhiteTemperatureLight? 
Is it normal ?

Reply:
Yes , devices like LTG002 have this type ( do not ask to me why :/ )

Reply:
![image](https://user-images.githubusercontent.com/1528694/99816154-95c21c00-2b4b-11eb-8177-09f507b06089.png)

Reply:
I think I know why , if you change temperature of white bulb , this color change ( yellowish / whitish )